src/FSharp.Control.TaskSeq/TaskSeq.fsi

```diff
@@ -215,14 +215,55 @@ type TaskSeq =
 
     /// <summary>
     /// Combines the given task sequence of task sequences and concatenates them end-to-end, to form a
```

Comment:
"the given task sequence of task sequences" feels like a mouthful, and Combines does not really help (Flattens would be a great one word TL;DR intro to my mind if I didnt know what concat did)
If I was writing something random from scratch: 

Yields a task sequence that unrolls each task sequence emanating from the source, concatenating them end to end,